functions/_common/urlTools.ts

```diff
@@ -0,0 +1,12 @@
+export function getGrapherFilters(
+    searchParams: URLSearchParams
+): Record<string, unknown> {
```

Comment:
I think it's actually not possible to get anything but a `string` as one of the values here.